src/server/actions/org.actions.ts

```diff
@@ -24,6 +25,15 @@ export const upsertOrgAction = siAdminAction(async (org) => {
         if (duplicate) {
             throw new Error('Organization with this name already exists')
         }
+
+        const adminUserId = await getUserIdFromActionContext()
+        await findOrCreateOrgMembership({
+            userId: adminUserId,
+            slug: org.slug,
+            isAdmin: true,
+            isResearcher: false, // by default not a researcher. can be changed later
+            isReviewer: false, // by default not a reviewer. can be changed later
+        })
     }
```

Comment:
I don't think we want to set the SiAdmin as a member of newly created orgs, I think what we'll need to do is update the new org form to also ask for a user's information and setup their account here.  That doesn't need to be part of these fixes though

Reply:
yes, I suspected that. I know it's not directly associated with the Sentry reported problem (which I'm still investigating)

The issue card description is kinda wrong or misleading then. I've also asked [Iris for clarification there](https://openstax.atlassian.net/browse/OTTER-223?focusedCommentId=18981&atlOrigin=eyJpIjoiNjQyYTk0ODE0YmQ1NDc0Njg5NTcxN2M2ZGZiNzZlOTAiLCJwIjoiaiJ9)